### PR TITLE
tests: Provide a fallback for CARGO_BIN_EXE_ for earlier Rusts

### PR DESCRIPTION
**Pull request by jrose-signal**
Unblocks #623.

### CODE REVIEW

**Comment by jrose-signal** [comment]
Failure is unrelated to my change.

**Comment by petrochenkov** [comment]
The CI failure should be fixed in https://github.com/eqrion/cbindgen/pull/622.

File: tests/profile.rs [hunk 1]
```diff
@@ -40,7 +40,20 @@ fn build_using_bin(extra_args: &[&str]) -> tempfile::TempDir {
...
+        Some(path) => path.into(),
+        None => {
+            // We must be on an older version of Rust.
+            // Guess where cbindgen would be relative to OUT_DIR.
+            let mut path = PathBuf::from(env!("OUT_DIR"));
+            path.pop();
+            path.pop();
+            path.pop();
+            path.push("cbindgen");
+            path.into_os_string()
```
Comment (emilio) [inline tests/profile.rs:53]: This is not correct with cross-compilation or what not, I think, but as a best-effort thing should do.

**Review by emilio — APPROVED** [review]

**Review by emilio — APPROVED** [review]

File: tests/profile.rs (hunk 1 above)
Comment (jrose-signal) [reply tests/profile.rs:53]: It's at least no worse than what the other tests are doing.

**Pull request merged**